cli-plugins/apps/command-reference/workload_get.hbs.md

```diff
@@ -2,12 +2,12 @@
 
 `tanzu apps workload get` is a command used to retrieve information and status about a workload.
 
-You can view workload details at whenever. Some details are:
+Some of the workload details in the command output are as follows:
 
- - Workload name and type.
- - The source of the workload application.
- - The supply chain which took care of the workload.
- - The supply chain resources which interact with the workload. It also has the output of the resource stamped out by the supply chain
+ - Workload name, type and namespace.
+ - The source code used to build the workload.
+ - The supply chain which processed the workload.
+ - The specific resources within the supply chain which interacted with the workload and the stamped out resources associated with each of those interactions.
  - The delivery workflow that the application follows.
  - If there is any issue while deploying the workload and finally which *pods* the workload generates and the knative services related to the workload.
```

Comment:
- Any issues associated with deploying the workload
- The *pods* the workload generates.
- And when applicable, the knative services related to the workload.